testsuite/MDAnalysisTests/coordinates/test_memory.py

```diff
@@ -0,0 +1,140 @@
+from numpy.testing import raises
+
+import MDAnalysis as mda
+from MDAnalysisTests.datafiles import DCD, PDB_small
+from MDAnalysisTests.coordinates.base import (BaseReference,
+                                              assert_timestep_almost_equal)
+from MDAnalysis.coordinates.memory import MemoryReader
+from numpy.testing import assert_equal
+
+from unittest import TestCase
+
+
+class MemoryReference(BaseReference):
+    def __init__(self):
+        super(MemoryReference, self).__init__()
+        self.universe = mda.Universe(PDB_small, DCD)
+        self.trajectory = \
+            self.universe.trajectory.timeseries(self.universe.atoms)
+        self.n_atoms = self.universe.trajectory.n_atoms
+        self.n_frames = self.universe.trajectory.n_frames
+        self.topology = PDB_small
+        self.reader = mda.coordinates.memory.MemoryReader
+
+        self.first_frame = MemoryReader.MemoryTimestep(self.n_atoms)
+        self.first_frame.positions = self.trajectory[:,0,:]
+        self.first_frame.frame = 0
+
+        self.second_frame = MemoryReader.MemoryTimestep(self.n_atoms)
+        self.second_frame.positions = self.trajectory[:,1,:]
+        self.second_frame.frame = 1
+
+        self.last_frame = MemoryReader.MemoryTimestep(self.n_atoms)
+        self.last_frame.positions = self.trajectory[:,self.n_frames - 1,:]
+        self.last_frame.frame = self.n_frames - 1
+
+        self.jump_to_frame = self.first_frame.copy()
+        self.jump_to_frame.positions = self.trajectory[:,3,:]
+        self.jump_to_frame.frame = 3
+
+
+class TestArrayReader(TestCase):
```

Comment:
Looks like you are reimplementing [BaseReaderTest](https://github.com/MDAnalysis/mdanalysis/blob/ee9210f1477139d805cbb66affa51b14a0577b1c/testsuite/MDAnalysisTests/coordinates/base.py#L151). Instead you can inherit from it and extend it with the tests you have written here.